openpower/package/openpower-pnor/Config.in

```diff
@@ -154,6 +154,18 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
         help
             String used to define name of openpower targeting binary file, ecc protected
 
+config BR2_OCMBFW_FILENAME
+        string "Name of the OCMBFW original file"
+        default "ocmbfw.bin"
+        help
+            String used to define name of OBMC original file (before any processing occurs)
```

Comment:
*IF* you touch this file again, maybe just say this (ie add in "FW Update"): 
String used to define name of OBMC FW Update original file (before any processing occurs)

Reply:
Alright, updated.